internal/graph/dotgraph.go

```diff
@@ -479,7 +480,10 @@ func min64(a, b int64) int64 {
 func escapeForDot(in []string) []string {
 	var out = make([]string, len(in))
 	for i := range in {
-		out[i] = strings.ReplaceAll(strings.ReplaceAll(strings.ReplaceAll(in[i], `\`, `\\`), `"`, `\"`), "\n", `\l`)
+		out[i] = escapeStringForDot(in[i])
 	}
 	return out
 }
+func escapeStringForDot(str string) string {
+	return strings.ReplaceAll(strings.ReplaceAll(strings.ReplaceAll(str, `\`, `\\`), `"`, `\"`), "\n", `\l`)
+}
```

Comment:
I wasn't quite sure what to call this function. I needed another version of this which works on a single string at a time.

Very open to suggestions/requests if you have any other ideas for what to call this! :) Thanks.